CSharp/Singleton.cs

```diff
@@ -0,0 +1,33 @@
+﻿
+
+namespace SingletonPattern
```

Comment:
I am trying to keep examples as simple as possible, therefore I guess there is no need to put this code into seperate namespace. :)